app/assets/javascripts/discourse/app/components/sidebar/messages-section.js

```diff
@@ -0,0 +1,20 @@
+import { action } from "@ember/object";
+
+import GlimmerComponent from "discourse/components/glimmer";
+import Composer from "discourse/models/composer";
+import { getOwner } from "discourse-common/lib/get-owner";
+import { next } from "@ember/runloop";
+
+export default class SidebarMessagesSection extends GlimmerComponent {
+  @action
+  composePersonalMessage() {
+    const composerArgs = {
+      action: Composer.PRIVATE_MESSAGE,
+      draftKey: Composer.NEW_TOPIC_KEY,
+    };
+
+    next(() => {
```

Comment:
Just wondering why we need `next` here?

Reply:
I think I took this from the composer model but it doesn't seem to be required. I've removed it.